src/utils.ts

```diff
@@ -342,9 +342,9 @@ export const withBatchedUpdates = <
 export const nFormatter = (num: number, digits: number): string => {
   const si = [
     { value: 1, symbol: "b" },
```

Comment:
```suggestion
    { value: 1, symbol: "B" },
```

https://simple.wikipedia.org/wiki/Byte